### PR TITLE
elasticsearch-head-plugin for elasticsearch v. 1.x.

### PR DESCRIPTION
**Pull request by ghostik**
The PR for elasticsearch-head-plugin of elasticsearch v. 1.x. An instruction for docker build added to README file.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -95,7 +95,44 @@ module.exports = function(grunt) {
 					keepalive: true
 				}
 			}
-		}
+		},
+//deb package
+	deb_package: {
+	  options: {
+            maintainer: "Konstantin Tushakov ktu@tradeshift.com",
+            version: "1.0.1",
```
Comment (mkostrikin) [inline Gruntfile.js]: Why version `1.0.1` ?

**Comment by mkostrikin** [comment]
build completed
:floppy_disk: 
please check resulting package this ES 1.4.x and ES 1.7.x

**Comment by ghostik** [comment]
Tested with ES 1.4.2 and 1.7.5 versions.

File: Gruntfile.js [hunk 2]
```diff
@@ -95,7 +95,44 @@ module.exports = function(grunt) {
 					keepalive: true
 				}
 			}
-		}
+		},
+//deb package
+	deb_package: {
+	  options: {
+            maintainer: "Konstantin Tushakov ktu@tradeshift.com",
+            version: "1.9.0",
```
Comment (mkostrikin) [inline Gruntfile.js]: Previous head version was `1.0.0`, so, `1.0.1` should work.

Comment (ghostik) [inline Gruntfile.js]: Changed to previous value.

**Comment by ghostik** [comment]
@rodrigooliveira  сс: @jnt please review.

**Comment by rodrigodealer** [comment]
Looks good to me. Squash your commits and you're good to go. 💯

**Pull request merged**